src/components/Blog/BlogSidebar.jsx

```diff
@@ -87,96 +49,39 @@ const useStyles = makeStyles((theme) => ({
       fontSize: theme.spacing(4),
     },
   },
-
-  post: {
-    paddingLeft: theme.spacing(1),
-    borderLeft: `solid 3px ${theme.palette.secondary.main}`,
-    marginTop: theme.spacing(4),
-  },
-
-  postText: {
-    fontWeight: 'bold',
-    transitionDuration: '250ms',
-    transitionTimingFunction: 'ease-in-out',
-    color: `${theme.palette.secondary.contrastText}`,
-    '&:hover': {
-      color: `${theme.palette.secondary.main}`,
-    },
-  },
-
-  postDate: {
-    color: `${theme.palette.secondary.main}`,
-    textTransform: 'uppercase',
-  },
 }));
 
-const BlogSidebar = ({
-  trips,
-  allTrips,
-  search,
-  inputValue,
-}) => {
+const BlogSidebar = () => {
   const classes = useStyles();
+  const allPostNumber = useSelector(getAllPostNumber);
 
   return (
     <Grid item xs={12} md={3}>
-      <Typography variant="h4" className={classes.title}>Kategorie</Typography>
+      <Typography variant="h4" className={classes.title}>
+        Kategorie

```

Comment:
Idealnie byłoby mieć plik z translacjami do takich stringów.
Ale to opcjonalnie warto rzucić okiem. Można na osobnym branchu.
https://react.i18next.com/getting-started

Reply:
spoko rzecz tylko nie wiem czy to będzie potrzebne do tej strony, ale można dodać pod koniec

Reply:
do strony nie, ale generalnie warto, tym bardziej, że nie jest to duża rzecz